script/pre_commit/pre_commit.md

```diff
@@ -0,0 +1,6 @@
+# Git pre-commit
+
+### Installation
+Under peloton root directory, run `ln -s ../../script/pre_commit/pre-commit.sh .git/hooks/pre-commit` for once to install locally.
```

Comment:
I think `../../` path is wrong if you are saying that they should be in the root directory.